pkg/cli/demo_cluster.go

```diff
@@ -885,14 +886,16 @@ func (c *transientCluster) listDemoNodes(w io.Writer, justOne bool) {
 			// the demo.
 			fmt.Fprintf(w, "node %d:\n", nodeID)
 		}
-		// Print node ID and console URL. Embed the autologin feature inside the URL.
-		pwauth := url.Values{
-			"username": []string{c.adminUser.Normalized()},
-			"password": []string{c.adminPassword},
-		}
 		serverURL := s.Cfg.AdminURL()
-		serverURL.Path = server.DemoLoginPath
-		serverURL.RawQuery = pwauth.Encode()
+		if !demoCtx.insecure {
+			// Print node ID and console URL. Embed the autologin feature inside the URL.
```

Comment:
maybe worth a comment about why we do not do this for insecure

Reply:
Sure, done.